spec/integration/integration_spec.rb

```diff
@@ -7,7 +7,7 @@
 describe 'Packaging and running a project' do
   def isolated_run(dir, cmd)
     Dir.chdir(dir) do
-      Bundler.clean_system("PATH=$GEM_HOME/bin:$PWD/../../../tmp/hadoop-current/bin:$PATH HUMBOLDT_TEST_PROJECT_PATH=$PWD rvm $RUBY_VERSION@humboldt-test_project do #{cmd}")
+      Bundler.clean_system(cmd)
```

Comment:
I still think you need to prepend Hadoop to the path, and the Travis runs seem to agree with me.